packages/forms/src/UIForm/fields/Button/SingleButton.component.js

```diff
@@ -6,7 +6,7 @@ import classNames from 'classnames';
 export default function SingleButton({ className, id, onTrigger, schema }) {
 	const {
 		bsStyle,
-		disabled,
+		disabled = false,
```

Comment:
Any way to do this with defaultProps instead?

Reply:
we should not defaultProps is lot slower mecanism and is framework bound when this way is faster and manguage bound

Reply:
No we can't it's not a props but a property of a props. If `schema` is provided without `disabled` or `readOnly` property, they will be undefined, even with a default props defined

Reply:
:+1: